### DISCUSSION TITLE
Error Cloning Repository

### DISCUSSION THREAD

**Discussion by pinacate** [organicmaps/organicmaps#243] [Q&A]
Hello All,

I get a permissions error cloning the repository. I'm working on a Mac. Here's a snippet from my log. Much of the clone seems to work but its failing on "3party/glm..."

Any help appreciated!

Cloning into '/Users/fmcclain/Local Documents/OMAPS/omapsapp/tools/osmctools'...
remote: Enumerating objects: 9, done.        
remote: Counting objects: 100% (9/9), done.        
remote: Compressing objects: 100% (7/7), done.        
remote: Total 34 (delta 1), reused 8 (delta 1), pack-reused 25        
Receiving objects: 100% (34/34), 168.73 KiB | 446.00 KiB/s, done.
Resolving deltas: 100% (14/14), done.
Cloning into '/Users/fmcclain/Local Documents/OMAPS/omapsapp/tools/twine'...
remote: Enumerating objects: 2, done.        
remote: Counting objects: 100% (2/2), done.        
remote: Compressing objects: 100% (2/2), done.        
remote: Total 2479 (delta 0), reused 1 (delta 0), pack-reused 2477        
Receiving objects: 100% (2479/2479), 689.22 KiB | 803.00 KiB/s, done.
Resolving deltas: 100% (1423/1423), done.
Cloning into '/Users/fmcclain/Local Documents/OMAPS/omapsapp/3party/glm'...
Warning: Permanently added the RSA host key for IP address '192.30.255.113' to the list of known hosts.
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

**Comment by biodranik** [comment]
Thanks! #241 should fix it.

**Comment by pinacate** [comment]
Ah, I replaced my git public key. That together with #241 fixed the problem. 
Thanks

**Reply by pinacate** [reply]
Great job everyone! 

After I fixed my public key/git problem the other steps went smoothly and I successfully build the Mac app.

**Comment by rtsisyk** [comment]
@pinacate Could you please mark this problem as resolved? Thanks.
